src/pages/PublishedSchedule/PublishedSchedule.jsx

```diff
@@ -56,7 +56,7 @@ const PublishedSchedule = () => {
 
     };
     renderTable();
-  }, [currentUser, curSeason, allSeasons]);
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'curSeason' and 'selectedSeason'. Either include them or remove the dependency array.